commands/sns/command.go

```diff
@@ -206,6 +233,33 @@ func snsTopicARNToTopicName(topicARN *string) *string {
 	return &split[len(split)-1]
 }
 
+// return true if topic should be skipped
+func updateDateToDelete(topicArn *string) (bool, error) {
+	snsTags, err := snsSvc.ListTagsForResource(&sns.ListTagsForResourceInput{ResourceArn: topicArn})
+	if err != nil {
+		return false, err
+	}
+	for _, tag := range snsTags.Tags {
+		if tag.Key != nil && *tag.Key == CheckTagNameUpdateDate {
+			if tag.Value == nil {
+				return false, nil
+			}
+			dataTime, err := time.Parse("2006-01-02T15:04:05.000-03:00", *tag.Value)
+			if err != nil {
+				return false, err
+			}
+			if dataTime.After(*rootArgs.UnusedSinceDate) {
+				l.Infof("%s is not old (%s) enough to delete", *topicArn, dataTime.String())
```

Comment:
This is not how structured logging works.

```go
l.Infow("topic is not old enough to delete", "topicArn", *topicArn, "date", dataTime.String())
```

Reply:
done! 👍